lib/zabbix/zabbix.go

```diff
@@ -203,3 +203,53 @@ func (a *API) GetTriggersInfo() (TriggerResults, error) {
 	}
 	return r.TriggerResults, nil
 }
+
+// AcknowledgeEvent acknowledges a single event and leaves a message.
+// It returns the event ID and an error if any.
+func (a *API) AcknowledgeEvent(eventID, message string) (int, error) {
+	params := struct {
+		EventIDs string `json:"eventids"`
+		// Event update action(s). This is bitmask field, any combination of values are acceptable.
+		// Possible values:
```

Comment:
It might be helpful to make these constant with comment and of a custom type. That way you can easily restrict possible values and it doesn't become one of those magic numbers whose meaning get's lost in time.

```go
type acknowledgementAction int

const (
	// Close acknowledgeds the event as closed
	Close acknowledgementAction = 1
	// Acknowledge ......
	Acknowledge acknowledgementAction = 2
	// Add ...
	AddMessage     acknowledgementAction = 4
	ChangeSeverity acknowledgementAction = 8
)```